regression-test/suites/compaction/test_table_level_compaction_policy.groovy

```diff
@@ -0,0 +1,131 @@
+    assertTrue(result.toString().containsIgnoreCase('"time_series_compaction_goal_size_mbytes" = "2048"'))
+
+    sql """
+        alter table ${tableName} set ("time_series_compaction_file_count_threshold" = "6000")
+        """
+
+    result = sql """show create table ${tableName}"""
+    logger.info("${result}")
+    assertTrue(result.toString().containsIgnoreCase('"time_series_compaction_file_count_threshold" = "6000"'))
+
+     sql """
+        alter table ${tableName} set ("time_series_compaction_time_threshold_seconds" = "3000")
+        """
+
+    result = sql """show create table ${tableName}"""
+    logger.info("${result}")
+    assertTrue(result.toString().containsIgnoreCase('"time_series_compaction_time_threshold_seconds" = "3000"'))
+
+    sql """ DROP TABLE IF EXISTS ${tableName} """
+
+    test {
+        sql """
+            CREATE TABLE ${tableName} (
+                    `c_custkey` int(11) NOT NULL COMMENT "",
+                    `c_name` varchar(26) NOT NULL COMMENT "",
+                    `c_address` varchar(41) NOT NULL COMMENT "",
+                    `c_city` varchar(11) NOT NULL COMMENT ""
+            )
+            DUPLICATE KEY (`c_custkey`)
+            DISTRIBUTED BY HASH(`c_custkey`) BUCKETS 1
+            PROPERTIES (
+                    "replication_num" = "1",
+                    "compaction_policy" = "ok"
+             );
+        """
+        exception "compaction_policy must be time_series or size_based"
+    }
+
+    test {
+        sql """
+            CREATE TABLE ${tableName} (
+                    `c_custkey` int(11) NOT NULL COMMENT "",
+                    `c_name` varchar(26) NOT NULL COMMENT "",
+                    `c_address` varchar(41) NOT NULL COMMENT "",
+                    `c_city` varchar(11) NOT NULL COMMENT ""
+            )
+            DUPLICATE KEY (`c_custkey`)
+            DISTRIBUTED BY HASH(`c_custkey`) BUCKETS 1
+            PROPERTIES (
+                    "replication_num" = "1",
+                    "time_series_compaction_goal_size_mbytes" = "1024"
+             );
+        """
+        exception "only time series compaction policy support for time series config"
+    }
+
+    test {
+        sql """
+            CREATE TABLE ${tableName} (
+                    `c_custkey` int(11) NOT NULL COMMENT "",
+                    `c_name` varchar(26) NOT NULL COMMENT "",
+                    `c_address` varchar(41) NOT NULL COMMENT "",
+                    `c_city` varchar(11) NOT NULL COMMENT ""
+            )
+            DUPLICATE KEY (`c_custkey`)
+            DISTRIBUTED BY HASH(`c_custkey`) BUCKETS 1
+            PROPERTIES (
+                    "replication_num" = "1"
+             );
+        """
+
+        sql """
+            alter table  ${tableName} set ("compaction_policy" = "ok")
+            """
+        exception "Table compaction policy only support for time_series or size_based"
+    }
+    sql """ DROP TABLE IF EXISTS ${tableName} """
+}
```

Comment:
add a test case for no compaction_policy config and check its output

Reply:
done